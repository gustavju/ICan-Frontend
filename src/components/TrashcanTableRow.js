import React from 'react';

const TrashcanTableRow = ({ trashcan, selectTrashcan }) => (
    <div onClick={(e) => selectTrashcan(e, trashcan.trashcanId)} className="trashcan-table__row">
        <div>{trashcan.trashcanId}</div>
        <div>{trashcan.TrashcanHistoryEntry.trashLevel}%</div>
        <div>{trashcan.TrashcanHistoryEntry.temperature}Â°C</div>
    </div>
)

export default TrashcanTableRow;