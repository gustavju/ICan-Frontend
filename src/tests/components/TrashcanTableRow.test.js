import React from 'react';
import { shallow } from 'enzyme';
import TrashcanTableRow from '../../components/TrashcanTableRow';
import trashcans from '../fixtures/trashcans';


test('should render the TrashcanTableRow', () => {
    const wrapper = shallow(<TrashcanTableRow trashcan={trashcans[0]} />);
    expect(wrapper).toMatchSnapshot();
});