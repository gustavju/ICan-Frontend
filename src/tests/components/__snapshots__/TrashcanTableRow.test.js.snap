// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the TrashcanTableRow 1`] = `
<div
  className="trashcan-table__row"
  onClick={[Function]}
>
  <div>
    8c13a125-cb3f-4a81-8cc5-c64574b0761f
  </div>
  <div>
    ONFIRE
  </div>
  <div>
    100.0
    %
  </div>
  <div>
    128.0
    Â°C
  </div>
  <div>
    household
  </div>
</div>
`;
